common/src/main/java/com/wynntils/features/ui/PlayerInfoFeature.java

```diff
@@ -0,0 +1,252 @@
+import com.wynntils.utils.render.type.TextShadow;
+import com.wynntils.utils.render.type.VerticalAlignment;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import net.minecraft.ChatFormatting;
+import net.minecraft.client.gui.components.PlayerTabOverlay;
+import net.minecraft.client.multiplayer.PlayerInfo;
+import net.minecraft.client.renderer.MultiBufferSource;
+import org.joml.Math;
+
+@ConfigCategory(Category.OVERLAYS)
+public class PlayerInfoFeature extends Feature {
+    private static final Comparator<PlayerInfo> PLAYER_INFO_COMPARATOR =
+            Comparator.comparing(playerInfo -> playerInfo.getProfile().getName(), String::compareToIgnoreCase);
+
+    public static final int DISTANCE_BETWEEN_CATEGORIES = 87;
+
+    public static final int ROLL_WIDTH = 27;
+
+    public static final int HALF_WIDTH = 178;
+
+    public static final int WIDTH = HALF_WIDTH * 2;
+
+    public static final int TOTAL_WIDTH = WIDTH + ROLL_WIDTH * 2;
+
+    public static final int MAX_LENGTH = 73;
+
+    @OverlayInfo(renderType = RenderEvent.ElementType.GUI)
+    public final PlayerInfoOverlay playerInfoOverlay = new PlayerInfoOverlay();
+
+    private class PlayerInfoOverlay extends Overlay {
+        @RegisterConfig
+        public final Config<Integer> openingDuration = new Config<>(125);
+
+        private double animationProgress = 0;
+        private long lastTime = -1;
+
+        protected PlayerInfoOverlay() {
+            super(
+                    new OverlayPosition(0, 0, VerticalAlignment.TOP, LEFT, OverlayPosition.AnchorSection.TOP_MIDDLE),
+                    TOTAL_WIDTH,
+                    Texture.PLAYER_INFO_OVERLAY.height());
+        }
+
+        private List<StyledText> lastPlayers = new ArrayList<>();
+        private long nextExecution = 0;
+
+        private List<StyledText> getAvailablePlayers() {
+            if (System.currentTimeMillis() < nextExecution && !lastPlayers.isEmpty()) return lastPlayers;
+
+            nextExecution = System.currentTimeMillis() + 250;
+
+            PlayerTabOverlay defaultTabList = McUtils.mc().gui.getTabList();
+
+            lastPlayers = McUtils.player().connection.getListedOnlinePlayers().stream()
+                    .sorted(PLAYER_INFO_COMPARATOR)
+                    .limit(80)
+                    .map(defaultTabList::getNameForDisplay)
+                    .map(StyledText::fromComponent)
+                    .filter(styledText -> !styledText.contains("§l"))
+                    .map(StyledText::getString)
+                    .map(styledText -> wrapText(styledText, MAX_LENGTH))
+                    .map(styledText -> styledText.replace("§7", "§0"))
+                    .map(StyledText::fromString)
+                    .toList();
+
+            return lastPlayers;
+        }
+
+        private static String wrapText(String input, int maxLength) {
+            if (McUtils.mc().font.width(input) <= maxLength) return input;
+
```

Comment:
Since this method does not actually wrap text onto the next line, can you rename the method? Can you also rename maxLength to maxWidth, since length implies the number of characters and width is being checked here? Lastly, this would probably be a better fit in StringUtils instead of a feature.